Character.h

```diff
@@ -10,43 +10,50 @@
 struct Character
 {
     Character(int hp, int armor_, int attackDamage_ );
+
     virtual ~Character() { }
     
     /*
-     a pure virtual getName function.
-     derived class stores the name, not the base class.
-     */
+    a pure virtual getName function.
+    derived class stores the name, not the base class.
+    */
     
     virtual const std::string& getName() = 0;
+
     virtual std::string getStats() = 0;
     
     virtual void attack( Character& other );
+
     void defend();
     
     void help( Character& other );
     
     int takeDamage(int damage);
     
-    int getHP() const { return hitPoints; }
-    int getArmorLevel() const { return armor; }
-    int getAttackDamage() const { return attackDamage; }
-    bool getIsDefending() const { return isDefending; }
-    
+    int getHP() const;
+
+    int getArmorLevel() const;
+
+    int getAttackDamage() const;
+
+    bool getIsDefending() const;
```

Comment:
Why did you change this?

Reply:
Had moved member function implementations to cpp file instead of header file, is reverted back to the header file.